### DISCUSSION TITLE
Fork and PR

### DISCUSSION THREAD

**Discussion by simplytoast1** [nVuln/homebridge-lg-thinq#64] [General]
I was able to do a fork of the code…

I assume I need to modify something here… https://github.com/simplytoast1/homebridge-lg-thinq/blob/master/src/v1/transforms/RefState.ts

Installing from the CLI is not an issue.  Just looking to pull the temperature sensors as it messes with my themostats and never modified.  I would like to know when the door is open or I need to enable express mode.

**Comment by nVuln** [comment]
Just confirm, you need remove thermostat temperature and express mode button?

**Comment by simplytoast1** [comment]
Just the temperature :)

**Reply by nVuln** [reply]
it here: https://github.com/simplytoast1/homebridge-lg-thinq/pull/1
